scripts/release-tools/operator-upgrade-test-helper.py

```
@@ -0,0 +1,395 @@
+            EC.presence_of_element_located((By.TAG_NAME, "body"))
+        )
+
+        shadow_content = None
+
+        if wait_for_selector:
+            try:
+                WebDriverWait(driver, 15).until(
+                    EC.presence_of_element_located(
+                        (By.CSS_SELECTOR, wait_for_selector))
+                )
+            except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)